src/sdk-reference/cpp/1/getting-started/snippets/document.cpp

```diff
@@ -0,0 +1,53 @@
+#include <iostream>
+
+#include "kuzzle.hpp"
+#include "websocket.hpp"
+
+using kuzzleio::Kuzzle;
+
+#define K_INDEX_NAME "nyc-open-data"
```

Comment:
Unnecessary define, made the code unclear to me

Reply:
@etrousset

Reply:
I don't agree with this, I think any constant parameter should be named using a define. It a very bad habit to use magic constant across a code, and we have to show a clean code to our users.

Reply:
Yes I agree for constant parameters but for us the index name is not a constant like a version number, it's a variable that can change across subsequent calls to the SDK method. Maybe we can define two string variable at the beginning of the main instead. However I will not fight for it in these snippets so do as you want